### PR TITLE
Update chart: labels, ServiceAccount, custom links

### PR DESCRIPTION
**Pull request by yurrriq**
This PR includes several updates we've been using for a while now. Let me know if you'd like me to split it up or change anything.

### CODE REVIEW

**Comment by hjacobs** [comment]
Can you summarize the changes for me in text? I'm not using Helm and I'm with merging the changes if this works for others.

**Comment by yurrriq** [comment]
Missed this till now...

Following community best practices:
- Use a named template for labels to avoid typos, ensure uniformity, etc
- Parameterize the service account to allow for using a preexisting one, etc

Add support for custom links, by reorganizing the ConfigMap and volume mounts to add `links.yaml` alongside `priving.csv`.
